ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
I am still confused; Let me ask a different why: isn't there one
obfuscator?  



Danny Colligan <notifications@github.com> writes:
> In ga4gh/datamodel/__init__.py:
> 
> > @@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
> >      """
> >      fields = DatasetCompoundId.fields + ['variantSet']
> >      containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
> > +    obfuscators = [DatasetCompoundId, None]
> 
> We refer to objects by their ids. All those ids within our system are
> obfuscated. If we want to lookup any item by an id, we need the correct
> (obfuscated) id.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>